examples/grpc_echo_sample/README.md

```diff
@@ -0,0 +1,16 @@
+# Echo Sample
+This sample is a simple echo function served as a simple grpc server
```

Comment:
can you write something in here about the function creation config settings for grpc, im thinking just a copy of the function register/create json to refer to (we prob need to do this for all examples so don't block on it if its annoying)

Reply:
Just use the UI? No need to add the extra comments here IMO.